perfkitbenchmarker/providers/azure/azure_virtual_machine.py

```diff
@@ -216,18 +278,34 @@ def GetLocalDisks(self):
 
 class DebianBasedAzureVirtualMachine(AzureVirtualMachine,
                                      linux_virtual_machine.DebianMixin):
-  DEFAULT_IMAGE = UBUNTU_IMAGE
+
+  # Example: ('b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-'
+  #           '14_04_3-LTS-amd64-server-20150908-en-us-30GB')
+  DEFAULT_IMAGE_PATTERN = re.compile(
+      r'b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-'
+      r'([0-9]+)_([0-9]+)(?:_([0-9]+))?'
```

Comment:
Yes. It appears as '...Ubuntu-14_04-LTS...'. The extracted tuple begins (14, 4, None, ...).